src/spinner/_spinner.scss

```diff
@@ -185,13 +185,21 @@
 }
 
 .mdl-spinner__circle-clipper {
-  display: inline-block;
+  display: block;
```

Comment:
I'm worried about potential backwards-compatibility issues with this change. Is the fix below compatible with us keeping this as an `inline-block`?